### PR TITLE
Feature/#34 find and last position of element in sorted array

### PR DESCRIPTION
**Pull request by Amoonguses1**
# binary search
Please check my code.

### CODE REVIEW

**Comment by Amoonguses1** [comment]
I have changed my code in order to follow pep8. Please review my commits..
@tkkawa @Ryuju0911

File: 34_find_first_and_last_position_of_element_in_sorted_array.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
+#time: O(logN)
```
Comment (tkkawa) [inline 34_find_first_and_last_position_of_element_in_sorted_array.py:1]: What is N?

File: 34_find_first_and_last_position_of_element_in_sorted_array.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+#space O(1)
+from typing import List
+
+
+class Solution:
+    def searchRange(self, nums: List[int], target: int) -> List[int]:
+        left = 0
+        n = len(nums)
```
Comment (tkkawa) [inline 34_find_first_and_last_position_of_element_in_sorted_array.py:9]: n is unclear.

File: 34_find_first_and_last_position_of_element_in_sorted_array.py [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+        right = n
+        exist_flag =True
+        indices_list= [-1,-1]
+        if n <= 0:
+            return indices_list
+        #find min
+        while left < right:
+            mid =(left + right) // 2
```
Comment (tkkawa) [inline 34_find_first_and_last_position_of_element_in_sorted_array.py:17]: Put whitespace after an operator.

**Review by tkkawa — COMMENTED** [review]
Revise the same points as #10. You should check the coding style.
And please submit the solution using [Meguru style](https://twitter.com/meguru_comp/status/697008509376835584).

**Comment by Amoonguses1** [comment]
I have fixed some points, unclear variables and inserting white space. I don't follow Meguru style because such style could be TLE. When target doesn't exist, computer runs the first while loop which is aimed to find minimum index eternally. That is caused the fact variables left never be equal to variables right. Avoiding the case, conditional expression changes into "left < right - 1". But, this is against Meguru style.  At least, I couldn't device any solutions which are accepted following Meguru style.
@tkkawa

**Comment by tkkawa** [comment]
Using the condition "left < right - 1" is not against Meguru style. So you can use it. In this case, you have to consider the initial values of left and right.
And the advantage of using the Meguru style is that left and right do not require +1 or -1 when updating. Therefore, the form of the conditional expression can be anything, but basically, you should use the above form.

**Comment by Amoonguses1** [comment]
Thank you for your explanations. I maybe understand strong points of Meguru style. I fixed my code following the style.
@tkkawa

**Review by tkkawa — APPROVED** [review]
LGTM

**Pull request merged**